src/sr_entities_handler.erl

```diff
@@ -170,3 +200,17 @@ atom_to_method(patch) -> <<"PATCH">>;
 atom_to_method(put) -> <<"PUT">>;
 atom_to_method(post) -> <<"POST">>;
 atom_to_method(delete) -> <<"DELETE">>.
+
+-spec method_to_atom(binary()) -> atom().
+method_to_atom(<<"GET">>) -> get;
+method_to_atom(<<"PATCH">>) -> patch;
+method_to_atom(<<"PUT">>) -> put;
+method_to_atom(<<"POST">>) -> post;
+method_to_atom(<<"DELETE">>) -> delete.
+
+-spec compose_handler_name(atom(), atom()) -> atom().
```

Comment:
What do you need the first argument of this function for if you'll only accept `handle`?
If you decide to keep it, you should change the spec to reflect that.